filebeat/input/udp/config.go

```diff
@@ -1,20 +1,26 @@
 package udp
 
 import (
+	"time"
+
 	"github.com/elastic/beats/filebeat/harvester"
+	"github.com/elastic/beats/filebeat/inputsource/udp"
 )
 
 var defaultConfig = config{
 	ForwarderConfig: harvester.ForwarderConfig{
 		Type: "udp",
 	},
-	MaxMessageSize: 10240,
-	// TODO: What should be default port?
-	Host: "localhost:8080",
+	Config: udp.Config{
+		MaxMessageSize: 10240,
+		// TODO: What should be default port?
+		Host: "localhost:8080",
```

Comment:
Perhaps there should not be a default and make the user explicitly state on which port to listen. The logstash udp input does this.

Reply:
I agree with that, but can only do in for 7.0 since this could break existing configuration.

Reply:
Note, the `TODO` was there before, my changes. :)

Reply:
+1 on the proposal from Andrew. Sorry about the TODO ...

Reply:
Is the udp input feature listed as experimental or beta? This could give us some leeway to make the change now.

Reply:
Lets do it in another PR, with a clear intend in the changelog.